server/mem_agent/venvstacks_manager.py

```
@@ -0,0 +1,416 @@
+
+        Args:
+            command: Command and arguments to pass to venvstacks
+            check: Whether to raise on non-zero exit
+
+        Returns:
+            CompletedProcess result
+        """
+        full_cmd = [sys.executable, "-m", "venvstacks"] + command
+        logger.info(f"Running: {' '.join(full_cmd)}")
+
+        result = subprocess.run(
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)